src/main/java/uk/gov/hmcts/sptribs/caseworker/event/page/IssueFinalDecisionSelectRecipients.java

```diff
@@ -22,14 +22,19 @@ public void addTo(PageBuilder pageBuilder) {
         pageBuilder
             .page("issueFinalDecisionSelectRecipients", this::midEvent)
             .pageLabel("Select recipients")
-            .label("LabelRecipients","Who should receive this decision notice?")
+            .label("labelIssueFinalDecisionSelectRecipients", "Who should receive this decision notice?")
             .complex(CaseData::getCicCase)
             .readonly(CicCase::getRepresentativeCIC, ALWAYS_HIDE)
             .done()
-            .complex(CaseData::getCaseIssueFinalDecision)
-            .optional(CaseIssueFinalDecision::getRecipientSubjectCIC, "")
-            .optional(CaseIssueFinalDecision::getRecipientRepresentativeCIC,"cicCaseRepresentativeCICCONTAINS \"RepresentativeCIC\"")
-            .optional(CaseIssueFinalDecision::getRecipientRespondentCIC,"")
+            .complex(CaseData::getCicCase)
+            .readonlyWithLabel(CicCase::getFullName, " ")
+            .optional(CicCase::getNotifyPartySubject, "")
```

Comment:
How come we are using an existing field to represent  this?  Don't we need a new field to represent who should be issued the final decision?

Reply:
it is for just sending notifications, there is no need for new fields and storing them